website/static/js/fileViewTreebeard.js

```diff
@@ -63,11 +63,13 @@ function FileViewTreebeard(data) {
             this.options.showTotal = Math.floor(containerHeight / this.options.rowHeight) + 1;
             this.redraw();
         },
-        lazyLoadOnLoad: function(tree) {
+        lazyLoadOnLoad: function(tree, event) {
```

Comment:
@caneruguz, does it still work for you when you have a github repo configured? `githubFangornConfig` also calls `Fangorn.Utils.scrollToFile` so I think that may be why it's not working for me.